schema.graphql

```diff
@@ -7,9 +7,9 @@ type Event @entity {
   moduleId: String! @index(unique: false)
   eventId: String! @index(unique: false)
   attributesTxt: String!
-  eventArg_0: String @index(unique: false)
-  eventArg_1: String @index(unique: false)
-  eventArg_2: String @index(unique: false)
```

Comment:
Is it OK to remove the index?

Reply:
We have to remove the index or we won't be able to process large extrinsics like a big batch.

By default they will be BTree which has a limit of 2713 bytes, so we'd get errors about that.

If the need to be indexed then we can add it into `compat.sql` and use a GIN or Gist (Not sure which one is best).

Right now we have indexes in that file for `left(event_arg_0, 100)` on these columns, which I think is a carry over from tooling.